policy/modules/admin/usbguard.te

```diff
@@ -0,0 +1,66 @@
+policy_module(usbguard, 1.0)
+
+## <desc>
+##	<p>
+##	Determine whether usbguard-daemon can modify its rules in
+##	/etc/usbguard.
+##	</p>
+## </desc>
+gen_tunable(usbguard_modify_rule_files, false)
+
+gen_require(`
+	type var_log_t;
+	type var_t;
+        type sysfs_t;
+        class dir { add_name create getattr open read remove_name rmdir search setattr write };
+        class file { create getattr map open read setattr unlink write };
+        class filesystem getattr;
+        class lnk_file { getattr read };
+        class netlink_kobject_uevent_socket { bind create read setopt };
+        class unix_stream_socket { accept listen };
```

Comment:
These aren't necessary, since `policy_module` already requires them for you.

Reply:
fixed